testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -2231,6 +2232,366 @@ fn stx_delegate_btc_integration_test() {
     channel.stop_chains_coordinator();
 }
 
+#[test]
+#[ignore]
+fn stack_stx_burn_op_test() {
```

Comment:
Can you help me understand the intended differences between this test and the `stack_stx_burn_op_integration_test()` function?

Reply:
I'll respond because I know Marzi was somewhat following the test pattern I did for the vote-for-aggregate-key burn op PR.

Because the burn operations are handled in separate places in 2.x vs 3.0, it's important to test that it works in both scenarios. So that's why the two tests. In hindsight, it would probably be a lot more straightforward to just have one integration test that made these operations in both epochs.